setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
We don't need this file, it's meant to be executed when creating the connector using the template. You'll see that it actually removes itself in line 38 :)